scripts/util/data_loading.py

```diff
@@ -22,6 +34,14 @@ def load_yaml(input_yaml: Path) -> Dict:
         return yaml.safe_load(yaml_stream)
 
 
+def write_json(data: Dict, output_json: Path) -> None:
+    """
+    Write a dictionary to a json file
+    """
+    with open(output_json, "w", encoding="utf8") as json_stream:
```

Comment:
Minor: Writing to/reading from `Path` like objects can be done a bit more seamless:

```python
with output_json.open("w", encoding="utf8") as json_stream:
```

Thought it was worth mentioning in case you weren't aware.

Reply:
Thanks, no I was not aware. Will update it.

Reply:
Actually, sorry, I'll retain the current syntax so that it is the same if the file (output_json) is a string.
This is a major code change and I'd prefer that the syntax is consistent here.